build.gradle

```diff
@@ -29,9 +29,18 @@ repositories {
 }
 
 dependencies {
+	def mapstruct = "1.3.1.Final"
+
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.flywaydb:flyway-core'
+	implementation "org.mapstruct:mapstruct:${mapstruct}"
+	annotationProcessor "org.mapstruct:mapstruct-processor:${mapstruct}"
+	testAnnotationProcessor "org.mapstruct:mapstruct-processor:${mapstruct}"
+
 	runtimeOnly 'org.postgresql:postgresql'
+
+	testImplementation "com.h2database:h2"
```

Comment:
Need to mention - in our case testing with h2 is impossible: h2 does not support Enumerations from Postgres that we are using.